src/parser/transform/transform_ddl.cpp

```diff
@@ -223,8 +231,12 @@ std::vector<PropertyDefinitionDDL> Transformer::transformPropertyDefinitionsDDL(
         if (property->oC_Expression()) {
             defaultValue = transformExpression(*property->oC_Expression());
         } else {
+            LogicalType type;
+            if (!LogicalType::tryConvertFromString(dataType, type)) {
+                type = *LogicalType::ANY();
```

Comment:
Shouldn't we throw?

Reply:
No, we should check the user defined type at binding stage.